healthtools/manage.py

```diff
@@ -4,7 +4,7 @@
 from healthtools.views import mount_app_blueprints
 
 
-log = logging.getLogger('healthtools')
+log = logging.getLogger(__name__)
```

Comment:
Any reason why this has been changed?

Reply:
Although both lines would return the same value, '__name__' is more pythonic.
I have updated the code base with the above feedback.

Reply:
Any data on it being more "pythonic"? How do other projects approach this first/entry set up of logging?